internal/controller/directivebreakdown_controller.go

```diff
@@ -532,6 +525,10 @@ func (r *DirectiveBreakdownReconciler) populateStorageBreakdown(ctx context.Cont
 	}
 
 	dbd.Status.Storage.AllocationSets = allocationSets
+
+	if wordList, present := argsMap["requires"]; present {
+		dbd.Status.RequiredDaemons = strings.Split(wordList, ",")
```

Comment:
RequiredDaemons won't get set for "#DW persistentdw". Do we have any reason to limit copy-offload to jobdw only?

Reply:
I've forgotten how 'persistentdw` is used. I thought it just asks for the volume to be mounted, and had nothing to do with running a compute job.

Reply:
"create_persistent" creates the persistent file system on the Rabbit nodes, and it doesn't require any computes to do that. 

"persistentdw" mounts the existing persistent file system from the Rabbits onto compute nodes so it can be used by a job application. There is a user application run as part of this job just like in a jobdw. I can't think of any reason to prevent users from initiating copy-offload with this type of job.

"destroy_persistent" deletes the persistent file system on the Rabbit nodes and doesn't require any computes.

Reply:
ok, persistentdw is working now.